[CI] Task ':x-pack:test:smb-fixture:composeUp' failed

Seen twice today so far, first one here:

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+intake/424/console

```
15:39:40 [0m[91mERROR(<class 'samba.provision.ProvisioningError'>): Provision failed - ProvisioningError: Your filesystem or build does not support posix ACLs, which s3fs requires.  Try the mounting the filesystem with the 'acl' option.
15:39:40   File "/usr/lib/python2.7/dist-packages/samba/netcmd/domain.py", line 442, in run
15:39:40 [0m[91m    nosync=ldap_backend_nosync, ldap_dryrun_mode=ldap_dryrun_mode)
15:39:40   File "/usr/lib/python2.7/dist-packages/samba/provision/__init__.py", line 2172, in provision
15:39:40 [0m[91m    skip_sysvolacl=skip_sysvolacl)
15:39:40   File "/usr/lib/python2.7/dist-packages/samba/provision/__init__.py", line 1806, in provision_fill
15:39:40     names.domaindn, lp, use_ntvfs)
15:39:40   File "/usr/lib/python2.7/dist-packages/samba/provision/__init__.py", line 1558, in setsysvolacl
15:39:40     raise ProvisioningError("Your filesystem or build does not support posix ACLs, which s3fs requires.  "
15:39:40 [0m
15:39:40 Service 'fixture' failed to build: The command '/bin/sh -c /fixture/src/main/resources/provision/installsmb.sh' returned a non-zero code: 255
15:39:40 
15:39:40 FAILURE: Build failed with an exception.
15:39:40 > Task :x-pack:test:smb-fixture:composeUp FAILED
15:39:40 :x-pack:test:smb-fixture:composeUp (Thread[Daemon worker,5,main]) completed. Took 9.616 secs.
```

Second one later: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+intake/426/console

--- a/x-pack/test/smb-fixture/src/main/resources/provision/installsmb.sh
+++ b/x-pack/test/smb-fixture/src/main/resources/provision/installsmb.sh
@@ -21,7 +21,7 @@ cat $SSL_DIR/ca.pem >> /etc/ssl/certs/ca-certificates.crt
 
 mv /etc/samba/smb.conf /etc/samba/smb.conf.orig
 
-samba-tool domain provision --server-role=dc --use-rfc2307 --dns-backend=SAMBA_INTERNAL --realm=AD.TEST.ELASTICSEARCH.COM --domain=ADES --adminpass=Passw0rd
+samba-tool domain provision --server-role=dc --use-rfc2307 --dns-backend=SAMBA_INTERNAL --realm=AD.TEST.ELASTICSEARCH.COM --domain=ADES --adminpass=Passw0rd --use-ntvfs
 
 cp /var/lib/samba/private/krb5.conf /etc/krb5.conf
 

